src/WebJobs.Script/Content/Script/res.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+module.exports = () => {
+    var res = {};
+
+    res.status = res.sendStatus = (statusCode) => {
+        res.statusCode = statusCode;
+        return res;
+    };
+
+    res.send = res.json = (body) => {
```

Comment:
So - with single output res.end is great, because now I can automatically wire up context.done to it.

Debated on what to do for res.json as there's a bunch of logic for processing body type in HttpBinding.cs.  To avoid confusion & try to express intent I should set content-type to `application/json` anyway and then the c# webapi stuff can do its content negotiation.